doc/release/new-policy.rst

```diff
@@ -0,0 +1,342 @@
+
+        Development builds have suffixes made with ``$(git describe --always --long)-dev``:
+
+        ..  code-block:: text
+
+            MAJOR.MINOR.PATCH-describe-dev
+
+            2.10.2-149-g1575f3c07-dev
+            3.0.0-alpha1-14-gxxxxxxxxx-dev
+            3.0.0-entrypoint-17-gxxxxxxxxx-dev
+            3.1.2-5-gxxxxxxxxx-dev
+
+    Alpha version
+
+        Alpha version has some of the features planned in the release series.
+        It can be incomplete or unstable, and can break the backwards compatibility
+        with the previous release series.
+
+        Alpha versions are published for early adopters and developers of dependent components,
+        such as connectors and modules.
+
+        ..  code-block:: text
+
+            MAJOR.MINOR.PATCH-alphaN
+
+            3.0.0-alpha1
+            3.0.0-alpha2
+
+    Beta version
+
+        Beta version has all the features which are planned for the release series.
+        It is a good choice to start developing a new application.
+
+        Readiness of a feature can be checked in a beta version to decide whether to remove the feature,
+        finish it later, or replace it with something else.
+        A beta version can still have a known bug in the new functionality,
+        or a known degradation since the previous release series that affects a common use case.
+
+        ..  code-block:: text
+
+            MAJOR.MINOR.PATCH-betaN
+
+            3.0.0-beta1
+            3.0.0-beta2
+
+    Release candidate
+
+        Release candidate is used to fix bugs, mature the functionality,
+        and collect feedback before an upcoming release.
+        Release candidate has the same feature set as the preceding beta version
+        and doesn't have known bugs in typical usage scenarios
+        or degradations from the previous release series.
+
+        Release candidate is a good choice to set up a staging server.
+
+        ..  code-block:: text
+
+            MAJOR.MINOR.PATCH-rcN
+
+            3.0.0-rc1
+            3.0.0-rc2
+            3.0.1-rc1
+
+Release series lifecycle
+--------------------------
+
+Every release series goes through the following stages:
+
+..  contents::
+    :local:
+
+Early development
+~~~~~~~~~~~~~~~~~
+
+The early development stage goes on until the first :term:`major release <major release>`.
+Alpha, beta, and release candidate versions are published at this stage.
+
+The stage splits into two phases:
+
+1.  Development of a new functionality through alpha and beta versions.
+    Features can be added and, sometimes, removed in this phase.
+
+2.  Stabilization starts with the first release candidate version.
+    Feature set doesn't change in this phase.
+
+Support
+~~~~~~~
+
+The stage starts when the first release is published.
+The release series now is an object of only backward compatible changes.
+
+At this stage, all known security problems and all found
+degradations since the previous series are fixed.
```

Comment:
```suggestion
degradations since the previous series are being fixed.
```
If I understand the process correctly, "are fixed" might be confusing, as it sounds like everything has already been fixed